Assets/Scripts/Board/Graph.cs

```diff
@@ -0,0 +1,80 @@
+using UnityEngine;
+using System.Collections.Generic;
+
+public class Graph
+{
+    public enum Building
+    {
+        Settlement,
+        City,
+    }
```

Comment:
We should add `NONE` so that doing `public Building building;` defaults to NONE, since I think enum-typed objects always default to 0th indexed value. But check me on that before making change, as long as we can distinguish between `building` being null/none and actually being set/occupied.

Reply:
I'm fine with that. I had that originally, but removed for no real reason.